README.md

```diff
@@ -1 +1,70 @@
-# math-magicians
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
- [ ] Kindly edit your readme documentation following the template provided by Microverse

Reply:
Ok. Done :) thanks for the feedback.